src/classes/TDTM_DefaultConfig.cls

```diff
@@ -218,6 +218,11 @@ public without sharing class TDTM_DefaultConfig {
         handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
                 Class__c = 'TGRD_ValidateData_TDTM', Load_Order__c = 1, Object__c = 'Term_Grade__c',
                 Owned_by_Namespace__c = 'hed', Trigger_Action__c = 'BeforeInsert;BeforeUpdate'));  
+                
+        // Prevents changes to a Contact or Course Offering on a Course Connection when a Term Grade or an Attendance Event is present on the Course Connection.
+        handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
+                Class__c = 'CCON_PreventUpdate_TDTM', Load_Order__c = 3, Object__c = 'Course_Enrollment__c',
```

Comment:
@skosgi1 Also, per something I learned from Carlos last week, this can just be a Load Order of 1 unless we really need this to run after some other trigger.